### DISCUSSION TITLE
Attributes processing

### DISCUSSION THREAD

**Discussion by CoderMariusz** [appwrite/appwrite#6464] [Q&A] [answered]
### 👟 Reproduction steps

when i add atribute it stay in progress state

### 👍 Expected behavior

creatye atribte and can createded document

### 👎 Actual Behavior

cant created document attribute hold "progress" state.

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment, marked as answer]
During some periods throughout the day, administrative operations such as creating databases, collections, buckets, or attributes can take an hour or so to complete. Please wait until it is complete. If the operation is still processing after a few hours, you can create a support post on [Discord](https://appwrite.io/discord) with your project ID.

**Comment by bkaniset** [comment]
@stnguyen90 - I have waited a whole day! is there a problem here?

**Reply by Jayfesco1** [reply]
Yes it's still there

**Reply by stnguyen90** [reply]
As mentioned, please create a support post on [Discord](https://appwrite.io/discord) with your project ID.
